test/functional/basic/iwdp-e2e-specs.js

```diff
@@ -0,0 +1,39 @@
+// transpile:mocha
+
+import chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import { SAFARI_CAPS } from '../desired';
+import { initSession, deleteSession, MOCHA_TIMEOUT } from '../helpers/session';
+import B from 'bluebird';
+
+chai.should();
+chai.use(chaiAsPromised);
+
+if (process.env.REAL_DEVICE) {
+  describe('ios_webkit_debug_proxy: ', function () {
+    this.timeout(MOCHA_TIMEOUT);
+    let caps, driver;
+
+    beforeEach(async () => {
+      caps = Object.assign(SAFARI_CAPS);
+      caps.startIWDP = true;
+    });
+
+    afterEach(async () => {
+      await deleteSession();
+      await B.delay(2000);
+    });
+
+    it('Should start a Safari session if "caps.startIWDP === true"', async () => {
+      caps.startIWDP = true;
+      driver = await initSession(caps);
+      await driver.source().should.not.be.rejected;
+      await driver.quit();
+    });
+
+    it('Should not start a Safari session if "caps.startIWDP === false"', async () => {
+      caps.startIWDP = false;
+      await initSession(caps).should.be.rejected;
```

Comment:
maybe we should say something about how it's rejected, to ensure we're catching the right error:

```js
await initSession(caps).should.be.rejectedWith(/webview/);
```

or something quasi-unique